quickstarts/microsoft.machinelearningservices/machine-learning-workspace-cmk/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create an Azure Machine Learning service workspace.",
+  "description": "This deployment template specifies an Azure Machine Learning workspace, and its associated resources including Azure Key Vault, Azure Storage, Azure Application Insights and Azure Container Registry. The example shows how to configure Azure Machine Learning for encryption with a customer-managed encryption key.",
+  "summary": "This template creates an Azure Machine Learning service workspace with public network connectivity.",
+  "githubUsername": "denniseik",
+  "dateUpdated": "2021-09-15",
+  "environments": [
+    "AzureCloud"
```

Comment:
if this is only supported in AzureCloud why do we have allowedValue locations for AzureUSGovernment?